blog/content/post/2020-02-05-meta-pcengines-xen.md

```diff
@@ -0,0 +1,139 @@
+---
+title: 'XEN on the PC Engines apu2 - why you want to use it'
+abstract: 'Xen Project creates a software system that allows the execution of 
+multiple virtual guest operating systems simultaneously on a single physical machine.
+In this case, it is a PC Engines apu2 platform.'
+cover: /covers/3mdeb-logo_simple-color.png
+author: norbert.kaminski
+layout: post
+published: false
+date: 2020-02-05
```

Comment:
@Asiderr if you want to do local preview, change date to actual or older (just temporarily) and set `published` to `true`